app/views/experiments/new.html.haml

```diff
@@ -92,12 +93,18 @@
       .row
         .small-6.small-centered.columns
           - ws_algorithm_options = workers_scaling_options
+          - ws_plgrid_default = [t('experiments.workers_scaling.mode.plgrid_default'), :plgrid_default]
           - ws_manual = [t('experiments.workers_scaling.mode.manual'), :manual]
           - ws_automatic = [t('experiments.workers_scaling.mode.automatic'), :automatic ]
-          - workers_scaling_mode = (ws_algorithm_options == ''? [ws_manual] : [ws_manual, ws_automatic])
+          - workers_scaling_mode = []
+          - workers_scaling_mode << ws_plgrid_default if plgrid_default_enabled
+          - workers_scaling_mode << ws_manual
+          - workers_scaling_mode << ws_automatic unless ws_algorithm_options == ''
           = select_tag 'workers_scaling_mode_select', options_for_select(workers_scaling_mode, workers_scaling_mode.first)
+          #plgrid_default{:style => 'display: none'}
```

Comment:
Nie znam się, ale to ma być na tym poziomie wcięcia? Wygląda mi to jak analogiczne do linii 107 (ale nie odpalałem więc nie wiem)